webui/react/src/routes/utils.ts

```diff
@@ -66,10 +66,10 @@ export const openBlank = (url: string): void => {
   window.open(url, '_blank', windowOpenFeatures.join(','));
 };
 
-export const openCommand = (command: Command): void => {
+export const openCommand = (command: Command | CommandTask): void => {
   const url = waitPageUrl(command);
   if (!url) throw new Error('command cannot be opened');
-  openBlank(url);
+  openBlank(process.env.PUBLIC_URL + url);
```

Comment:
**suggestion:** Couple of recommendation:

- It looks like `command.url` may already be defined, so if using it if that's the case.
- Throwing an error seems a bit rough, since `openCommand` is not wrapped in a `try/catch` block. I think if it can't figure where to go, just do nothing.

```
export const openCommand = (command: Command | CommandTask): void => {
  const url = command.url || waitPageUrl(command);
  if (url) openBlank(process.env.PUBLIC_URL + url);
};
```

Reply:
i used handleError to leave some trace of this error but made it so it doesn't throw.